auth-web/src/components/auth/manage-business/action-buttons/ActionButtons.vue

```diff
@@ -0,0 +1,441 @@
+            true
+        )
+
+      } else if (isApproved && isOtherEntities(props.business)) {
+        // MVP redirect to COLIN
+        updateMainActionButton(
+            ActionButtonsTexts.Actions.DOWNLOAD_FORM,
+            redirectToFormsPage,
+            true,
+            ActionButtonsTexts.Tooltips.GO_TO_CORPORATE_ONLINE_REGISTER,
+            true
+        )
+
+      } else {
+        updateMainActionButton('Open Name Request', openManageNR)
+
+      }
+    } else {
+      // it is not invitation, nor name request ==> It is regular incorporation stuff
+      const isDraft = isTemporaryBusiness(props.business)
+      const isActive = entityStatus(props.business) === BusinessState.ACTIVE
+      const isHistorical = props.business?.status?.toUpperCase() === BusinessState.HISTORICAL.toUpperCase() // is this OK check?
+      const type = getType(props.business)
+      mainActionClickedEvent.name = 'HandleBusinessAction'
+
+      if (
+          // isDraft &&
+          [AffiliationTypes.INCORPORATION_APPLICATION as string, CorpTypes.INCORPORATION_APPLICATION as string].includes(type)) {
+        updateMainActionButton('Resume Draft', resumeDraft)
+
+      } else if (
+          // isDraft &&
+          [AffiliationTypes.REGISTRATION as string, CorpTypes.REGISTRATION as string].includes(type)) {
+        updateMainActionButton('Resume Draft', resumeDraft)
+
+      } else if (isActive && isModernizedEntity(props.business)) {
+        updateMainActionButton('Manage Business', openBusinessDashboard)
+
+      } else if (isActive && !isModernizedEntity(props.business)) {
+        updateMainActionButton(
+            'Manage Business',
+            redirectToColin,
+            true,
+            ActionButtonsTexts.Tooltips.GO_TO_CORPORATE_ONLINE_ACCESS,
+            true
+        )
+
+      } else if (isHistorical && isModernizedEntity(props.business)) {
+        updateMainActionButton('Manage Business', openBusinessDashboard)
+
+      } else if (isHistorical && !isModernizedEntity(props.business)) {
+        updateMainActionButton(
+            'Manage Business',
+            redirectToColin,
+            true,
+            ActionButtonsTexts.Tooltips.GO_TO_CORPORATE_ONLINE_ACCESS,
+            true
+        )
+
+      } else if ((isHistorical || isActive) && isSocieties(props.business)) {
+        updateMainActionButton('Manage Business', redirectToSocietiesOnline, true, 'Go to Societies Online to access this business', true)
+
+      } else if (isHistorical) {
+        updateMainActionButton(
+            'Manage Business',
+            redirectToColin,
+            true,
+            ActionButtonsTexts.Tooltips.GO_TO_CORPORATE_ONLINE_ACCESS,
+            true
+        )
+      } else {
+        //if all else fails ...
+        updateMainActionButton('Manage Business', openBusinessDashboard)
+      }
+    }
+
+    return {
+      mainActionButton
+    }
+  }
+})
+</script>
+
+<style scoped lang="scss">
+
+// Tooltips
+.v-tooltip__content {
```

Comment:
is this used anywhere else? perhaps better in a scss file? so we can use it across the app? if it's a one off.. disregard